src/Data/Int.purs

```diff
@@ -83,3 +86,47 @@ even x = x .&. 1 == 0
 -- | ```
 odd :: Int -> Boolean
 odd x = x .&. 1 /= 0
+
+-- | The number of unique digits (including zero) used to represent integers in
+-- | a specific base.
+newtype Radix = Radix Int
+
+-- | The base-2 system.
+binary :: Radix
+binary = Radix 2
+
+-- | The base-8 system.
+octal :: Radix
+octal = Radix 8
+
+-- | The base-10 system.
+decimal :: Radix
+decimal = Radix 10
+
+-- | The base-16 system.
+hexadecimal :: Radix
+hexadecimal = Radix 16
+
+-- | Create a `Radix` from a number between 2 and 36.
+radix :: Int -> Maybe Radix
+radix n | n >= 2 && n <= 36 = Just (Radix n)
+        | otherwise         = Nothing
```

Comment:
This is great. I forgot when you opened #17 originally, but the reason we don't have the radix functions already is because allowing arbitrary radix arguments to be used with the underlying JS function meant that they will throw exceptions when the radix is out of range, so doing this while providing common values is a nice way of solving that.